index.js

```diff
@@ -1,57 +1,63 @@
-module.exports = function(options) {
-  options = options || {};
-  options.features = options.features || {};
-  options.import = options.import || {};
-  options.import.name = options.import.name || 'default';
-
-  if (typeof options.import.module !== 'string') {
-    throw new Error("options.import.module must be the name of a module, e.g. 'my-app/features'");
-  }
+module.exports = function(babel) {
+  var t = babel.types;
 
-  Object.keys(options.features).forEach(function(feature) {
-    var value = options.features[feature];
-
-    if (typeof value === 'string') {
-      if (value === 'enabled') {
-        options.features[feature] = true;
-      } else if (value === 'disabled') {
-        options.features[feature] = false;
-      } else if (value === 'dynamic') {
-        options.features[feature] = null;
-      } else {
-        throw new Error("An unknown feature state '" + value + "' was detected for '" + feature + "'. Valid values are 'enabled', 'disabled' and 'dynamic'");
-      }
-    }
-  });
+  return {
+    visitor: {
+      Program: function(path, state) {
```

Comment:
I had to move the options validation into the visitor because the options don't seem to be available before that